utils/qemu/Makefile

```diff
@@ -266,6 +270,7 @@ PKG_CONFIG_DEPENDS += \
 	CONFIG_QEMU_UI_VNC_PNG \
 	CONFIG_QEMU_UI_VNC_SASL \
 	CONFIG_QEMU_UI_SPICE \
+	CONFIG_QEMU_DEV_USB \
 
 
 # QEMU configure script does not recognize these options
```

Comment:
Interesting that CONFIGURE_ARGS is used for a meson package...

Reply:
It still uses `./configure`. Do you think that it should be changed?